src/zLOG/tests/testzLog.py

```diff
@@ -41,12 +41,12 @@ def setUp(self):
         # tearDown() will close and remove all the handlers that pop
         # into existence after setUp() runs.  This breaks into internals,
         # but I couldn't find a sane way to do it.
-        self.handlers = logging._handlers.keys()  # capture current handlers
+        self.handlers = list(logging._handlers.keys())  # capture current handlers
```

Comment:
I prefer the `list(logging._handlers)` spelling because it's shorter, and because it avoids one unnecessary list copy on Python 2.

If anyone else feels that `list(d.keys())` is clearer, I won't argue.